packages/core/backend/sirius-components-collaborative/src/main/resources/schema/core.graphqls

```diff
@@ -1,8 +1,8 @@
 "Upload type used for the file upload"
 scalar Upload
 
-"Instant type used for dates"
-scalar Instant
+"Temporal type used for dates"
+scalar Temporal
```

Comment:
This scalar is used by others including downstream applications, you cannot remove it. It has been moved in core.graphqls on purpose, it is not specific to the Gantt representations. Changing something with regards to how dates are sent/received on the GraphQL layer would require an ADR to discuss this change and see how, if it is needed, we could change it.

Reply:
You 're totally right